Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -965,7 +966,7 @@ static std::optional<Vector<uint8_t>> unwrapCryptoKey(JSGlobalObject* lexicalGlo
 #if ASSUME_LITTLE_ENDIAN
 template <typename T> static void writeLittleEndian(Vector<uint8_t>& buffer, T value)
 {
-    buffer.append(reinterpret_cast<uint8_t*>(&value), sizeof(value));
+    buffer.append(std::span { reinterpret_cast<uint8_t*>(&value), sizeof(value) });
```

Comment:
You've used this pattern a few times in this patch. I wonder if a `toSpan<>` template would clean thing up a bit more?